### PR TITLE
fix for noetic

### PR DESCRIPTION
**Pull request by KamalanathanN**
reference : https://stackoverflow.com/questions/40691311/save-results-in-yaml-file-with-python

### CODE REVIEW

File: src/goal_sequence_visualizer.py [hunk 1]
```diff
@@ -35,8 +35,9 @@ def init_goal_sequence_visualizer(mat):
 		pose.orientation.z = quaternion[2]
 		pose.orientation.w = quaternion[3]
 		pose_array.poses.append(pose)
-		marker.header.frame_id = "/map"
+		marker.header.frame_id = "map"
 		marker.header.seq = i
+		marker.ns = ""
```
Comment (padhupradheep) [inline src/goal_sequence_visualizer.py]: Not required I guess! I will not have it!

File: src/goal_sequence_visualizer.py [hunk 2]
```diff
@@ -58,4 +59,4 @@ def init_goal_sequence_visualizer(mat):
 		pub, pub_txt, rate, pose_array, marker_array =	init_goal_sequence_visualizer(server.mat)
 		pub.publish(pose_array)
 		pub_txt.publish(marker_array)
-		rate.sleep()
\ No newline at end of file
+		rate.sleep()
```
Comment (padhupradheep) [inline src/goal_sequence_visualizer.py]: I will not try to remove the newline at the end, maybe in the future, there might be a problem with us for the version control

**Review by padhupradheep — CHANGES_REQUESTED** [review]
Thanks for the pull request. We do have a noetic branch now, please do check the reviews and give us a new PR.

File: src/goal_sequence_visualizer.py (hunk 1 above)
Comment (KamalanathanN) [reply src/goal_sequence_visualizer.py]: 🥲ops

**Review by padhupradheep — COMMENTED** [review]
Please leave a PR to the Noetic branch! Thanks!

**Pull request closed without merging**